tests/e2e.sh

```diff
@@ -6,34 +6,36 @@ cp ginkgolinter testdata/src/a
 cd testdata/src/a
 
 # no suppress
-[[ $(./ginkgolinter a/... 2>&1 | wc -l) == 2604 ]]
+[[ $(./ginkgolinter a/... 2>&1 | wc -l) == 2628 ]]
```

Comment:
Inspection of `succeed/succeed.go` and `succeedconfig/succeed.go` shows that 24 new errors should be reported. 8 of these assertions of the form `Expect(err).To(Succeed())` vanish when succeed suppression is enabled, leaving 16 extra.

So for each test case here, the number of errors should rise by either 24 or 16.